vtr_flow/scripts/vtr-flow.py

```diff
@@ -203,6 +226,23 @@ def vtr_command_main(arg_list, prog=None):
                                    verbose=True if args.verbosity > 2 else False,
                                    echo_cmd=True if args.verbosity >= 4 else False)
```

Comment:
Would be good to have a comment where verbosity is first defined that gives the legal range and meaning of various levels.

Reply:
These levels were previous functionality that the perl script does not implement so I changed it to match the perl script, which uses a boolean to either output the error description or not.